Makefile

```diff
@@ -58,8 +58,8 @@ endif
 
```

Comment:
@rinn7e the metadata generation need to be updated in the Haskell code.

Running `make metadata` now I see several things to remove:
1. `TZIP-12` is still listed as a supported interface
2. The `all_tokens` view, which is FA2-specific
3. The `token_metadata` view, which is also FA2-specific

Reply:
I forgot about this, pushed an update now.